test/e2e/windows/kubelet_stats.go

```diff
@@ -80,6 +80,11 @@ var _ = SIGDescribe("[Feature:Windows] Kubelet-Stats [Serial]", func() {
 						framework.ExpectEqual(*podStats.CPU.UsageCoreNanoSeconds > 0, true, "Pod stats should not report 0 cpu usage")
 						framework.ExpectEqual(*podStats.Memory.WorkingSetBytes > 0, true, "Pod stats should not report 0 bytes for memory working set ")
 
+						framework.ExpectEqual(podStats.Network != nil, true, "Pod stats should report network stats")
+						framework.ExpectEqual(podStats.Network.Name != "", true, "Pod stats should report network name")
+						framework.ExpectEqual(*podStats.Network.TxBytes > 0, true, "Pod stats should report network Tx stats")
+						framework.ExpectEqual(len(podStats.Network.Interfaces) > 0, true, "Pod Stats should report individual interfaces stats")
+
```

Comment:
We're expecting `Network` to be filled in now but there may be a nil deref for lines 84-86 if for some reason this ever changes right?

Reply:
There is a `nil` check on line 83 where the test would stop processing and report `network` was found to be `nil`

Reply:
Aha, didn't know what the behavior for ExpectEqual was (e.g. immediate failure). Feel free to resolve!